### ISSUE TITLE
Question about Regression loss

### ISSUE THREAD

**Issue by Sebo-the-tramp** [naver/mast3r#104]
First of all, thank you so much for this fantastic work!

I am exploring the loss functions and computations in detail. From the paper, the loss seems to be mostly computed on the normalized point-cloud values. Both predictions and gt are normalized to the same space. On the other hand, the "Regr3D_ScaleShiftInv" confuses me because it is only used in testing. Does it mean that the normalization process in training doesn't inherently scale predictions gt to be aligned even in scale? Is this behavior used to encourage the model to learn to predict the correct scale as well, or am I missing some crucial details?

Thank you so much.

**Comment by dshatwell23** [comment]
@Sebo-the-tramp did you ever find the answer? I'm also curious about this. 

The README file is also a bit confusing. Why is Regr3D_ScaleShiftInv used in the "Demo" section, but not in "Our Hyperparameters". 

If I understand this correctly, in the get_all_pts3d function (mast3r.losses.Regr3D), the variable "gt_scale" is set to True during inference. Does this mean that the ground truth point maps are not normalized?
